### ISSUE TITLE
Alt text missing from icon links on Team page

### ISSUE THREAD

**Issue by c-deems** [alaoweb/2021.alaoweb.org#187]
Some of the icons for Twitter, personal websites, etc. do not seem to contain alt text, which could be a problem for screen readers.
